core/src/main/java/com/graphhopper/util/exceptions/ConnectionNotFoundException.java

```diff
@@ -25,7 +30,75 @@
  * @author Peter Karich
  */
 public class ConnectionNotFoundException extends DetailedIllegalArgumentException {
+    private static final String INTRO = "Connection between ";
+    private static final String FMT = "point %d and point %d";
+    private static final String JOIN = ", and ";
+    private static final String OUTRO = " not found";
```

Comment:
Why define them here explicitely?

Reply:
I'm using `INTRO` and `JOIN` multiple times (lines 54 and 57 respectively) hence I figured to store the strings in class variables. `FMT` and `OUTRO` exist to handle the string creation in `unconnectedListString`  consistently (i.e. no magic string components). 

How would you prefer this to be handled?